modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+To remove the File Integrity Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* -> File Integrity Operator page.
```

Comment:
Should File Integrity Operator be bold as the name of the page?
```suggestion
. Navigate to the *Operators* -> *Installed Operators* -> *File Integrity Operator* page.
```

Reply:
@GroceryBoyJr Sorry, I was asking if _File Integrity Operator_ should be bold, as it is the name of the page.